src/scripts/underground/Mine.ts

```diff
@@ -1,10 +1,12 @@
 class Mine {
+    public static sizeX: number = 25;
+    public static sizeY: number = 12;
     public static grid: Array<Array<KnockoutObservable<number>>>;
     public static rewardGrid: Array<Array<any>>;
     public static itemsFound: KnockoutObservable<number>;
     public static itemsBuried: number;
     public static rewardNumbers: Array<number>;
-    public static toolSelected: KnockoutObservable<GameConstants.MineTool> = ko.observable(GameConstants.MineTool["Chisel"]);
+    public static toolSelected: KnockoutObservable<Mine.Tool> = ko.observable(0);
```

Comment:
The `0` here seems a little opaque.
```suggestion
    public static toolSelected: KnockoutObservable<Mine.Tool> = ko.observable(Mine.Tool.Chisel);
```

Reply:
Yeah there was a problem here.
`Mine.Tool.Chisel` can't be used before its declaration in the namespace, and the namespace cannot come before the class it is merged with. I'm not sure of a better solution but I'll add a comment here

Reply:
Yeah there was a problem here.
`Mine.Tool.Chisel` can't be used before its declaration in the namespace, and the namespace cannot come before the class it is merged with. I'm not sure of a better solution but I'll add a comment here